initializr-docs/src/test/java/io/spring/initializr/stub/InitializrIntegrationTests.java

```diff
@@ -5,34 +5,32 @@
 
 import org.junit.Test;
 import org.junit.runner.RunWith;
-
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.boot.test.context.TestConfiguration;
 import org.springframework.boot.web.client.RestTemplateBuilder;
 import org.springframework.cloud.contract.stubrunner.StubFinder;
-import org.springframework.cloud.contract.stubrunner.spring.AutoConfigureStubRunner;
+import org.springframework.cloud.contract.wiremock.AutoConfigureWireMock;
 import org.springframework.context.annotation.Bean;
+import org.springframework.core.env.Environment;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 import org.springframework.http.RequestEntity;
 import org.springframework.http.ResponseEntity;
 import org.springframework.test.context.junit4.SpringRunner;
 import org.springframework.web.client.RestTemplate;
 
-import static org.assertj.core.api.Assertions.*;
-import static org.springframework.boot.test.context.SpringBootTest.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
 
 // tag::test[]
 @RunWith(SpringRunner.class)
 @SpringBootTest(webEnvironment = WebEnvironment.NONE)
-@AutoConfigureStubRunner(
-		ids = "io.spring.initializr:initializr-web",
-		workOffline = true)
+@AutoConfigureWireMock(stubs = "classpath:/META-INF/io.spring.initializr/initializr-web/**/*.json", port = 0)
 public class InitializrIntegrationTests {
```

Comment:
The test does not work from IJ simply because the generated directory is not on the classpath. Even though the test works fine with Maven, I am not keen to promote and document something that doesn't work from the IDE. I guess we miss a maven config in `initializr-web`? If you had to add the directory manually we can document that but I guess I prefer to wait for the other solution then.